github-git/0-first-contribution.md

```diff
@@ -0,0 +1,188 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+myst_html_meta:
+    "title": "Intro to Collaborative GitHub: Make your first contribution to open source"
+    "description lang=en": "Learn the steps involved with using GitHub to collaborate and contribute to open source code and documentation. A beginner-friendly guide."
+    "keywords": "GitHub, OpenSource, beginner-friendly"
+    "property=og:locale": "en_US"
+    "og:image": /images/github/steps-to-contribute.png
+    "og:image:alt": An image that shows the steps for contributing to open source on GitHub.
+---
+
+# Mastering GitHub Collaboration Skills 
```

Comment:
@sneakers-the-rat, have a look at this version and see what you think. it's a different approach - but it gives the learner the steps first and then it sends them to pages with more info. (if they need it). i just don't know how this will land with people.